rollup.config.js

```diff
@@ -3,27 +3,24 @@ import commonjs from 'rollup-plugin-commonjs';
 import nodeResolve from 'rollup-plugin-node-resolve';
 import replace from 'rollup-plugin-replace';
 import uglify from 'rollup-plugin-uglify';
+import { dependencies } from './package.json'
 
 const env = process.env.NODE_ENV;
 const config = {
   entry: 'src/index.js',
   plugins: [],
 };
 
+const externals = Object.keys(dependencies).join('|');
+
 if (env === 'es' || env === 'cjs') {
   config.format = env;
   config.sourceMap = true;
-  config.external = [
-    'invariant',
-    'lodash.curry',
-    'lodash.isfunction',
-    'lodash.isobject',
-    'lodash.isplainobject',
-    'lodash.map',
-    'redux',
-  ];
+  config.external = id => RegExp(`^(${externals})(\/.*)?$`).test(id);
```

Comment:
Is there any reason this is not just `config.external = Object.keys(dependencies)`?

Reply:
because then it would not externalize `package/foo` https://rollupjs.org/#peer-dependencies

Reply:
Ahh gotcha, that makes sense.